velox/docs/functions/window.rst

```diff
@@ -0,0 +1,80 @@
+================
+Window functions
+================
+
+Window functions perform calculations across rows of the query result. They
+run after the HAVING clause but before the ORDER BY clause.
+
+Invoking a window function requires special syntax using the OVER clause to specify
```

Comment:
Since Velox doesn't support SQL interface, this statement is a bit strange. Would it be possible to modify it somehow to take into account that Velox is a library.

Reply:
Hmm... This is an interesting point. I had used the SQL interface as it clarifies the window function definition. I have revised the writeup with the context that Velox is a library.

We can write a developer article on authoring a window function at some later time if required.